packages/earth-admin/src/components/error-boundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { ErrorMessages } from 'components/error-messages';
+
+export class ErrorBoundary extends React.Component {
```

Comment:
why isn't this a function component?

Reply:
There's currently no way to create an Error Boundary as a function component: https://reactjs.org/docs/error-boundaries.html